server/game/gameSystems/MoveCardSystem.ts

```diff
@@ -16,11 +25,12 @@ import * as Contract from '../core/utils/Contract';
  *
  * @property destination - The target zone for the card. Excludes discard pile, space arena, ground arena, and resources.
  * @property shuffle - Indicates whether the card should be shuffled into the destination.
- * @property bottom - Indicates whether the card should be placed at the bottom of the destination.
+ * @property shuffleTargets - Indicates whether all targets should be shuffled before added into the destination.
```

Comment:
I think the person writing the card doesn't think in terms of targets, maybe this should be called shuffleCards or shuffleMovedCards?